bin/www

```diff
@@ -92,7 +92,7 @@ function onListening() {
     const bind = typeof addr === 'string' ? 'pipe ' + addr : 'port ' + addr.port;
     debug(`App now running on ${bind}`);
 
-    if (!!process.env.ENABLE_BROWSER_SYNC === true) {
+    if (process.env.ENABLE_BROWSER_SYNC === 'true') {
```

Comment:
Was trying to turn browser sync off locally and realised that setting to false was still enabling the feature. Explicitly checking for a 'true' value rather than coercing. Not sure if this is me missing something here.